products-dashboard/products-dashboard.dash.yaml

```diff
@@ -0,0 +1,47 @@
+global:
```

Comment:
Please format this file in kubersmarts or in VSCode:

* Open the code 
* Hold: `Ctrl + Shift` and press `i`
It should correctly indent this file